src/main/scala/amba/apb/ToTL.scala

```diff
@@ -50,11 +51,11 @@ class APBToTL()(implicit p: Parameters) extends LazyModule
 
 
       val beat = TransferSizes(beatBytes, beatBytes)
-      // The double negative here is to work around Chisel's broken implementation of widening ~x.
-      val aligned_addr =  ~(~in.paddr | (beatBytes-1).U)
+      //TODO: The double negative here is to work around Chisel's broken implementation of widening ~x.
+      val aligned_addr =  ~in.paddr
```

Comment:
Is this the same as before? Not familiar with chisel widening on `~`. @sequencer